hax/hax/message.py

```diff
@@ -47,6 +47,7 @@ class ProcessEvent(BaseMessage):
 @dataclass
 class BroadcastHAStates(BaseMessage):
     states: List[HAState]
+    is_broadcast_local: bool
```

Comment:
Can you add a one-line comment telling what this field is about?
BTW doesn't it make sense just create two different message types (instead of a single one) if the semantics differs much?

Reply:
So basically we want to have 2 types of broadcasts, local and to entire cluster. So Idea is that when a user module needs to broadcast an event `is_local_broadcast` is set to false so the event is notified to all the nodes in the cluster. And when the message is processed as part of EQ (`is_broadcast_local=False`) the respective nodes broadcast the event locally using `is_broadcast_local=True`.
I thought of having another message type as well but amount of changes did not justify the need. We anyway need to broadcast, just need to differentiate between local vs global. But if this makes the code unreadable then we can have a separate message type.

Reply:
Understood. Let's keep this boolean flag `is_broadcast_local` for now; we can revisit the need of separate class later.

Just please add an explanatory comment to the code to give an idea what this flag is about.
Thanks!

Reply:
Sure.